app/cli/cmd/policy_develop_eval.go

```diff
@@ -74,7 +74,7 @@ evaluates the policy against the provided material or attestation.`,
 	cobra.CheckErr(cmd.MarkFlagRequired("material"))
 	cmd.Flags().StringVar(&kind, "kind", "", fmt.Sprintf("Kind of the material: %q", schemaapi.ListAvailableMaterialKind()))
 	cmd.Flags().StringSliceVar(&annotations, "annotation", []string{}, "Key-value pairs of material annotations (key=value)")
-	cmd.Flags().StringVarP(&policyPath, "policy", "p", "policy.yaml", "Path to custom policy file")
+	cmd.Flags().StringVarP(&policyPath, "policy", "p", "policy.yaml", "Policy reference (file://, http://, https://, chainloop://)")
```

Comment:
chainloop:// can be the default one, so we might not need ot provide it.

Reply:
in fact I do not think we should use chainloop:// as default.

Do you need to provide file:// always now?